crates/core/tedge_agent/src/agent.rs

```diff
@@ -225,10 +225,24 @@ impl SmAgent {
         })
     }
 
+    pub fn get_log_dir(&self) -> PathBuf {
```

Comment:
```suggestion
    pub fn log_dir(&self) -> PathBuf {
```

Getter functions are normally called only like the variable they're accessing. So no `get_` prefix! :wink: